src/app/components/site/phy/RoutesPhy.tsx

```diff
@@ -179,9 +179,13 @@ export const RoutesPhy = [
 
     // Legacy Routes
     <Redirect key={key++} exact from="/mission" to="/about" />,
-    <Redirect key={key++} exact from="/boards" to="/my_gameboards" />,
+    <Redirect key={key++} exact from="/boards" to={PATHS.MY_GAMEBOARDS} />,
+    <Redirect key={key++} exact from="/my_gameboards" to={PATHS.MY_GAMEBOARDS} />,
     <Redirect key={key++} exact from="/game_builder" to={PATHS.GAMEBOARD_BUILDER} />,
+    <Redirect key={key++} exact from="/gameboard_builder" to={PATHS.GAMEBOARD_BUILDER} />,
+    <Redirect key={key++} exact from="/add_gameboard/:id" to={`${PATHS.ADD_GAMEBOARD}/:id`} />,
     <Redirect key={key++} exact from="/board/:id" to={`${PATHS.GAMEBOARD}#:id`} />,
+    <Route key={key++} exact path="/gameboards" render={(props) => <Redirect to={`${PATHS.GAMEBOARD}${props.location.hash ?? ""}`}/>} />, // Route wrapper needed to handle hash
```

Comment:
I think this is _slightly_ cleaner, but I understand that you liked neither approach!
```suggestion
    <Redirect key={key++} exact from="/gameboards" to={{pathname: PATHS.GAMEBOARD, hash: window.location.hash}} />,
```